docs/language/learn-ql/introduction-to-ql.rst

```diff
@@ -3,7 +3,7 @@ Introduction to QL
 
 QL is the powerful query language that underlies CodeQL, which is used to analyze code.
 Queries written with CodeQL can find errors and uncover variants of important security vulnerabilities.
-Visit Semmle's `security research page <https://lgtm.com/security>`__ to read about examples of vulnerabilities that we have recently found in open source projects.
+Visit `GitHub Security Lab <https://securitylab.github.com/>`__ to read about examples of vulnerabilities that we have recently found in open source projects.
```

Comment:
In a few places we also link to the Semmle blog. I guess these will have to change to link to the research page on the Security Lab site at some point.